--- a/lib/platform/std/src/impl/codec/opus.zig
+++ b/lib/platform/std/src/impl/codec/opus.zig
@@ -30,7 +30,7 @@
         application: opus.Application,
         frame_ms: u32,
     ) !Self {
-        var inner = try opus.Encoder.init(allocator, sample_rate, channels, application);
+        const inner = try opus.Encoder.init(allocator, sample_rate, channels, application);
         return .{ .inner = inner, .frame_ms = frame_ms, .alloc = allocator };
     }
 
@@ -78,7 +78,7 @@
         channels: u8,
         frame_ms: u32,
     ) !Self {
-        var inner = try opus.Decoder.init(allocator, sample_rate, channels);
+        const inner = try opus.Decoder.init(allocator, sample_rate, channels);
         return .{ .inner = inner, .frame_ms = frame_ms, .alloc = allocator };
     }
 
